prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -56,10 +57,13 @@ def build_agent(
         temperature=0.7,
     )
     agent = Agent(llm=LLM(generator=generator), engine=engine)
+    agent.max_tries = 3
```

Comment:
Default `max_tries` is 10, which is excessive (leaves the user waiting a long time before feedback)